save-orchestrator-common/src/main/kotlin/com/saveourtool/save/orchestrator/service/ContainerService.kt

```diff
@@ -85,31 +81,22 @@ class ContainerService(
                     interval = configProperties.agentsStartCheckIntervalMillis.milliseconds,
                     numberOfChecks = configProperties.agentsStartTimeoutMillis / configProperties.agentsStartCheckIntervalMillis
                 ) {
-                    areAgentsHaveStarted.computeIfAbsent(executionId) { AtomicBoolean(false) }.get()
+                    !orchestratorAgentStatusService.containsAnyByExecutionId(executionId)
```

Comment:
Also, can we rename this method is some manner it, to point out that we address here to our collection and not to `db`?

Reply:
I wanted to highlight it by prefix `orchestrator`. Can rename it to `agentStatusInMemoryRepository`

Reply:
renamed the class and field name